content/numpy/concepts/math-methods/terms/around/around.md

```diff
@@ -0,0 +1,69 @@
+| Field | Value |
+| --- | --- |
+| Title | .around() |
+| Description | Rounds each element in an array to the given number of decimals. |
+| Subjects | Computer Science, Data Science |
+| Tags | Arrays, Methods, NumPy, Rounding |
+| CatalogContent | learn-python-3, paths/computer-science |
```

Comment:
```suggestion
| Field | Value |
| --- | --- |
| Title | .around() |
| Description | Rounds each element in an array to the given number of decimals. |
| Subjects | Computer Science, Data Science |
| Tags | Arrays, Methods, NumPy, Rounding |
| CatalogContent | learn-python-3, paths/computer-science |
```
Remove all the vertical lines over here. 
 Title: 
Description: 
Subjects:
-
The correct format should be like this.